src/main/resources/liquibase/crop_changelog/21_1_0.xml

```diff
@@ -22,4 +22,80 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="mderamos" id="v21.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="program_location_default"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create program_location_default table
```

Comment:
Hi @mderamos 
I have a concern, is there a reason to create a new table to store default locations that are unique per program? Should not be more accurate to create columns y workbench_project? 
I think this simplify the code